src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.stories.js

```diff
@@ -1799,6 +1799,35 @@ const exampleInputText = {
   ],
 };
 
+const exampleInputTime = {
+  $schema: 'http://adaptivecards.io/schemas/adaptive-card.json',
+  type: 'AdaptiveCard',
+  version: '1.0',
+  body: [
+    {
+      type: 'Input.Time',
+      id: 'timeValueSet',
+      label: 'value:set',
```

Comment:
let's have something that looks more like a message intended for a user to read. Maybe "Start time" and "End time"